### ISSUE TITLE
Failing to find art: FakeUserAgentError

### ISSUE THREAD

**Issue by natervader** [desbma/sacad#79]
Sacad's been giving me trouble finding artwork for about a week now, and can't determine what's causing the issue. Could've simply been something on my part but I don't believe it was.

Installed through the AUR's `sacad` script. I have also tried the `sacad-git` git build with the same outcome.

System info:
OS: Arch Linux
Kernel: 6.0.7
Terminal: Kitty

Not sure what else would be relevant. 

Here's a sample output with what I normally feed it:
```
Analyzing library: 1dir [00:00, 87.86dir/s, files=7, albums=1, missing covers=1, errors=0]
Searching covers: 100%|███████████████████████████████████████████| 1/1 [00:00<00:00,  1.05cover/s, ok=0, errors=1, no result found=0]
Traceback (most recent call last):
  File "/usr/lib/python3.10/site-packages/fake_useragent/fake.py", line 135, in __getattr__
    return random.choice(self.data_browsers[browser_name])
KeyError: 'firefox'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/bin/sacad_r", line 33, in <module>
    sys.exit(load_entry_point('sacad==2.7.0', 'console_scripts', 'sacad_r')())
  File "/usr/lib/python3.10/site-packages/sacad/recurse.py", line 458, in cl_main
    get_covers(work, args)
  File "/usr/lib/python3.10/site-packages/sacad/recurse.py", line 367, in get_covers
    asyncio.get_event_loop().run_until_complete(root_future)
  File "/usr/lib/python3.10/asyncio/base_events.py", line 649, in run_until_complete
    return future.result()
  File "/usr/lib/python3.10/site-packages/sacad/recurse.py", line 300, in post_download
    status = future.result()
  File "/usr/lib/python3.10/site-packages/sacad/__init__.py", line 47, in search_and_download
    cover_sources = [cls(*source_args) for cls in source_classes]
  File "/usr/lib/python3.10/site-packages/sacad/__init__.py", line 47, in <listcomp>
    cover_sources = [cls(*source_args) for cls in source_classes]
  File "/usr/lib/python3.10/site-packages/sacad/sources/amazoncd.py", line 39, in __init__
    super().__init__(*args, base_domain=urllib.parse.urlsplit(self.base_url).netloc, **kwargs)
  File "/usr/lib/python3.10/site-packages/sacad/sources/amazonbase.py", line 19, in __init__
    self.current_ua = self.ua.firefox
  File "/usr/lib/python3.10/site-packages/fake_useragent/fake.py", line 138, in __getattr__
    raise FakeUserAgentError(
fake_useragent.errors.FakeUserAgentError: Error occurred during getting browser: firefox
```

I did some light digging around the associated files and what little I was able to figure out on my own was it using a dummy browser to grab the images, however it doesn't appear to be finding any browsers at all. I do have Firefox and Chromium installed if it makes a difference.

**Comment by natervader** [comment]
Also noticed [this thread](https://github.com/desbma/sacad/issues/46) about a similar sounding issue, and it didn't seem to have any effect. Getting the same output.

**Comment by desbma** [comment]
This is a bug from [fake-useragent](https://github.com/fake-useragent/fake-useragent), try removing the cache: `rm -r ~/.cache/sacad/fake_useragent`

**Comment by natervader** [comment]
> This is a bug from [fake-useragent](https://github.com/fake-useragent/fake-useragent), try removing the cache: `rm -r ~/.cache/sacad/fake_useragent`

That did the trick! Thanks a bunch desbma.

**Comment by desbma** [comment]
I have [changed the cache path to be version specific](https://github.com/desbma/sacad/commit/e482e5383fa951beda8287988fc590fb14255f16), so manual action should no longer be needed to fix this starting with sacad 2.7.1.
